index.js

```diff
@@ -0,0 +1,47 @@
+let fs = require('fs');
+let solc = require('solc');
+let path = require('path');
+
+let contractCache;
+
+let WeTrustContract = {
+  /**
+   * Call this function upon server start. It reads and compiles the contract, and saves it in a cache variable.
+   * Throws if there was an error.
+   */
+  init: function(done) {
```

Comment:
Just to understand better: you mean moving the logic of init to rosca-mvp? If so, why do we need getContract all? It only returns a var.